frame/merkle-mountain-range/src/benchmarking.rs

```diff
@@ -30,15 +30,15 @@ benchmarks! {
 		let leaves = x as u64;
```

Comment:
for instantiable pallets, you should use the macro `benchmarks_instance_pallet` above

Reply:
I fixed it,

Note: it wasn't introduce by the PR, in old macro, ppl should use benchmarks_instance